app/src/main/java/ch/epfl/sweng/erpa/util/ActivityUtils.java

```diff
@@ -91,4 +102,20 @@ public static void setUsernameInMenu(NavigationView navigationView, UserProfile
             username.setText("currently not logged");
         }
     }
+
+    public static void setMenuInToolbar(AppCompatActivity activity, Toolbar toolbar){
+        activity.setSupportActionBar(toolbar);
+        ActionBar actionbar = activity.getSupportActionBar();
+        actionbar.setDisplayHomeAsUpEnabled(true);
+        actionbar.setHomeAsUpIndicator(R.drawable.ic_menu);
+    }
+
+    public static boolean onOptionItemSelectedUtils(int menuItemId, DrawerLayout mDrawerLayout) {
+        switch (menuItemId) {
```

Comment:
Pourquoi un `switch` ici au lieu d'un simple if si t'as qu'un seul cas ?

Reply:
because it's how the "super" function was made (the one i override in the activity) but i can change it.